.github/actions/functional-test-cucumber/action.yml

```diff
@@ -0,0 +1,19 @@
+name: Functional Tests
+description: Run Functional Tests
+runs:
+  using: "composite"
+  steps:
+    - run: npm install --silent --unsafe-perm
+      shell: bash
+    - run: npm pack
+      shell: bash
+    - run: tar xf kourou-$(node -e 'console.log(require("./package.json").version)').tgz
```

Comment:
There is a dedicated action to get package.json version https://github.com/marketplace/actions/get-current-package-version

Reply:
I think this is unnecessary, see my comment below